assembly/pom.xml

```diff
@@ -89,4 +78,41 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>targz</id>
+      <properties>
+        <assembly.format>tar.gz</assembly.format>
+      </properties>
+    </profile>
+
+    <profile>
+      <id>scala-2.10</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>${project.groupId}</groupId>
+          <artifactId>livy-repl_2.10</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+      </dependencies>
+    </profile>
+
+    <profile>
+      <id>scala-2.11</id>
+      <dependencies>
+        <dependency>
+          <groupId>${project.groupId}</groupId>
+          <artifactId>livy-repl_2.11</artifactId>
```

Comment:
I use profile `scala-2.11` and `scala-2.10` to differentiate the repl jar packaged in assembly. If `scala-2.11` is specified, the assembly module will depend on `repl_2.11` and package the dependencies into folder `repl-jars`. If not specified it will always pick `repl_2.10` jars into folder `repl-jars`.